### PR TITLE
plotjuggler: route name in window title

### PR DESCRIPTION
**Pull request by GregorKikelj**
Requires https://github.com/commaai/PlotJuggler/pull/50

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Do old versions of PlotJuggler crash or just ignore the new option? If it crashes, we'll have to detect this.

**Comment by sshane** [comment]
Older versions refuse to start: `Unknown option 'window_title'.`

**Comment by GregorKikelj** [comment]
I can run ./plotjuggler -h and check for the option being listed, if not auto reinstall or emit a warning

**Comment by sshane** [comment]
Maybe we can check PJ version? Assert a required or at least as high as version?

**Comment by adeebshihadeh** [comment]
Let's add a minimum version check and show a prompt when the minimum version isn't met.

**Comment by GregorKikelj** [comment]
Not sure how to check the version

**Comment by adeebshihadeh** [comment]
> Not sure how to check the version

`./plotjuggler -v`

**Comment by GregorKikelj** [comment]
Hmm ok not sure why I completely missed it

File: tools/plotjuggler/juggle.py [hunk 1]
```diff
@@ -46,6 +47,13 @@ def install():
       tar.extractall(path=INSTALL_DIR)
 
 
+def check_plotjuggler_version():
+  proc = subprocess.Popen([PLOTJUGGLER_BIN, "-v"], stdout=subprocess.PIPE)
+  (out, _) = proc.communicate()
+  version = out.decode("utf-8").split(" ")[1]
+  return version >= MINIMUM_PLOTJUGGLER_VERSION
```
Comment (adeebshihadeh) [inline tools/plotjuggler/juggle.py:54]: these are strings. it'll work properly if you split into a tuple

Comment (GregorKikelj) [reply tools/plotjuggler/juggle.py:54]: Works as strings as well, but can do tuples if prefered.

Comment (sshane) [reply tools/plotjuggler/juggle.py]: Not always:

```
"0.8.9" > "0.8.10"
Out[126]: True
```

File: tools/plotjuggler/juggle.py [hunk 2]
```diff
@@ -160,6 +169,9 @@ def juggle_route(route_or_segment_name, segment_count, qlog, can, layout, dbc=No
     print("PlotJuggler is missing. Downloading...")
     install()
 
+  if get_plotjuggler_version() < MINIMUM_PLOTJUGGLER_VERSION:
+    raise Exception(f"PlotJuggler version is too old. Update with './juggle.py --install'")
```
Comment (adeebshihadeh) [inline tools/plotjuggler/juggle.py:173]: should offer to update with a prompt

Comment (adeebshihadeh) [reply tools/plotjuggler/juggle.py]: on second thought, better to just install. that's what we do when it's missing

**Pull request merged**